tests/helper_functions.py

```diff
@@ -80,11 +80,16 @@ def score_model_in_sagemaker_docker_container(
     """
     env = dict(os.environ)
     env.update(LC_ALL="en_US.UTF-8", LANG="en_US.UTF-8")
+    port = get_safe_port()
+    scoring_cmd = (
+        f"mlflow deployments run-local -t sagemaker --name test -m {model_uri}"
+        f" -C image=mlflow-pyfunc -C port={port} --flavor {flavor}"
+    )
     proc = _start_scoring_proc(
-        cmd=["mlflow", "sagemaker", "run-local", "-m", model_uri, "-p", "5000", "-f", flavor],
```

Comment:
`mlflow sagemaker run-local` is obviated by `mlflow deployments run-local -t sagemaker` and is removed above, so we migrate this test helper function that relies on sagemaker local deployment to use the `mlflow deployments run-local` API